tests/jvm/src/test/scala/scala/meta/tests/semanticdb/AnnotationsSuite.scala

```diff
@@ -0,0 +1,76 @@
+package scala.meta.tests
+package semanticdb
+
+import scala.meta.internal.semanticdb.scalac._
+
+class AnnotationsSuite extends DatabaseSuite(SemanticdbMode.Slim) {
+
+  annotations(
+    """
+      |// Type annotation
+      |class A[@specialized(Int, Long) T](v: T)
+      |
+      |// Class annotation
+      |@deprecated("Use D", "1.0") class B { }
+      |
+      |object C {
+      |  // Variable annotation
+      |  @transient @volatile var m: Int = 0
+      |
+      |  // Expression annotation
+      |  def f(x: Option[String]) = (x: @unchecked) match { case Some(y) => y }
+      |}
+    """.stripMargin,
+    """
+      |_root_.scala.volatile#{
+      |  _root_.scala.annotation.meta.field#
```

Comment:
I feel like a `@` would help readability, wdyt?

Reply:
this is for testing. There is a `@` in `Denotation.toString`.